v22.1/create-schedule-for-backup.md

```diff
@@ -10,7 +10,7 @@ docs_area: reference.sql
 For more information about creating, managing, monitoring, and restoring from a scheduled backup, see [Manage a Backup Schedule](manage-a-backup-schedule.html).
 
 {{site.data.alerts.callout_info}}
-Core users can only use backup scheduling for [full backups](#create-a-schedule-for-full-backups-only-core) of clusters, databases, or tables.
+Core users can only use backup scheduling for [full backups](#create-a-schedule-for-full-backups-only-core) of clusters, databases, or tables. Scheduling backups without the `FULL BACKUP ALWAYS` clause will result in a warning. 
```

Comment:
I think this is a great place to put it, but perhaps make it more actionable? Maybe something like:

"If you do not specify the `FULL BACKUP ALWAYS` clause when you schedule a backup, you will receive a warning that the schedule will only run full backups."

Reply:
ditto this.